docs/source/developing/writing-docs.adoc

```diff
@@ -110,4 +183,7 @@ $ cd ..
 $ rake build
 ----
 
-If the build completes successfully, the site is available under _preview/openshift-origin/latest/welcome/index.html_.
+If the build completes successfully, the site is available under *_preview/openshift-origin/latest/welcome/index.html_*.
+
+OpenShift runs a nightly build in which they update the documentation. After you update the documentation,
```

Comment:
Sure, the problem is that it is plain wrong. It implies that you if you "update the documentation", somehow it magically get picked up by a nightly update. Neither is clear what "update the documentation" actually means, nor does it touch on the CentOS CI job which is the actual trigger for updating the docs on doc.openshift.org.